src/e2e/java/teammates/e2e/pageobjects/InstructorSearchPage.java

```diff
@@ -86,7 +86,7 @@ private WebElement getStudentTableForHeader(CourseAttributes course) {
         List<WebElement> studentCoursesResult = getStudentCoursesResult();
 
         return studentCoursesResult.stream().filter(studentCourse -> {
-            String courseHeader = studentCourse.findElement(By.className("card-header")).getText();
```

Comment:
@tau-bar The `test.search.active` attribute in `test.properties` needs to be set to `true` to run any search-related E2E tests locally, else the test will automatically pass.

After debugging, the reason for the test failing is as follows, which shows that `courseHeader` has an extra set of square brackets.
![image](https://user-images.githubusercontent.com/54243224/195552144-40a54024-68ca-4cf3-bb56-8e3201f99918.png)
Seems that this change was unintentionally added when reverting the E2E tests. Reverting changes means that the files should not show up under "Files changed" at all. In the future, do be more careful so as to avoid unnecessary time and pain spent on debugging :")

Reply:
Hm, this seems weird, because when I don't include the additional brackets in the test, I obtain this: 
![image](https://user-images.githubusercontent.com/61085398/195581276-f642c831-0706-4ebd-afad-a2ac9d2742f7.png).

I'll try and investigate why it's different.

Reply:
Yikes, this means there might be a difference in behaviour across different OSes, which opens a whole can of worms. For the purposes of this PR, let's stick to the automated check results; do create a separate issue to report the differences in E2E test behaviour on your local machine. Thanks a lot for your effort in investigating this!

Reply:
@zhaojj2209 For the purposes of reporting the issue, may I know what OS you were using to debug the E2E tests?

Reply:
@tau-bar I used Firefox on Ubuntu, which is the same environment as the automated E2E testing job on GitHub.